tests/views/test_blog_pages.py

```diff
@@ -0,0 +1,87 @@
+from tests import PMGLiveServerTestCase
+from tests.fixtures import dbfixture, PostData
+import flask
+
+from pmg.models import db, Post
+
+
+class TestBlogPages(PMGLiveServerTestCase):
+    def setUp(self):
+        super(TestBlogPages, self).setUp()
+
+        self.fx = dbfixture.data(PostData)
+        self.fx.setup()
+
+    def tearDown(self):
+        self.fx.teardown()
+        super(TestBlogPages, self).tearDown()
+
+    def test_blog_post_page(self):
+        """
+        Test blog post page (http://pmg.test:5000/blog/<slug>)
+        """
+        post = self.fx.PostData.the_week_ahead
+        self.get_page_contents("http://pmg.test:5000/blog/%s/" % post.slug)
+        self.assertIn(post.title, self.html)
+        self.assertIn(post.body[0:100], self.html)
```

Comment:
I tend to recommend hardcoding strings in the asserts rather than using what's in the fixture data programmatically since it'll be easier to notice the test becoming invalid if someone changes the value in the fixture to something like the empty string if it's hardcoded. It's very little duplication with a big win in robustness.

Reply:
I don't think you need to go and change all this now. If it's quick, that'd be nice but this is more for the future. I'll try and capture this in best practises as well.

Reply:
Ah ok. So like:

`self.assertIn("The Week Ahead: End of the First Term", self.html)`?